src/simtools/data_model/schema.py

```diff
@@ -0,0 +1,77 @@
+"""Module providing functionality to read schema."""
+
+from pathlib import Path
+
+import simtools.utils.general as gen
+from simtools.constants import MODEL_PARAMETER_METASCHEMA, MODEL_PARAMETER_SCHEMA_PATH
+
+
+def model_parameter_schema_files(schema_directory=MODEL_PARAMETER_SCHEMA_PATH):
```

Comment:
Suggest to rename the functions to make their purpose clearer. Maybe `get_parameters_and_schema_files`, also the following `get_schema_path_for_model_parameter`and `get_validated_schema_version`.

Reply:
OK - for me it appears clear the `schema.model_parameter_schema_files` gives me the schema files. But your comment shows that this is not the case for everyone, so I changed it and added everywhere the `get_`...